src/JustSaying/AwsTools/MessageHandling/IMessageRequester.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Amazon.SQS.Model;
+
+namespace JustSaying.AwsTools.MessageHandling
+{
+    public interface IMessageRequester
```

Comment:
`IMessageReceiver`? I'm thinking the name would be more the intended result, rather than the _how_ to get the result.